src/com/facebook/buck/apple/AppleCxxPlatforms.java

```diff
@@ -301,6 +301,12 @@ static AppleCxxPlatform buildWithExecutableChecker(
     Tool lldb =
         VersionedTool.of(getToolPath("lldb", toolSearchPaths, xcodeToolFinder), "lldb", version);
 
+    Tool plutil =
+        VersionedTool.of(
+            getToolPath("plutil", ImmutableList.of(Paths.get("/", USR_BIN)), xcodeToolFinder),
```

Comment:
`plutil` is a util that has been on osx since 10.2, but unlike all these other tools it is not shipped in Xcode. This seemed like the preferred way of getting it here.